src/execute.rs

```diff
@@ -131,12 +131,55 @@ pub fn execute_change_url(
     };
 
     tokens().replace(deps.storage, &token_id, Some(&updated_token), Some(&token))?;
+
     Ok(Response::new()
         .add_attribute("action", "change url")
         .add_attribute("token_id", token_id)
         .add_attribute("url", url))
 }
 
+pub fn execute_change_color(
```

Comment:
Since the code here is nearly identical to `execute_change_url` I suggest that we merge the two functions into one and use `Option` for each param.

Reply:
good idea!